test/fw/ptl/utils/pbs_procutils.py

```diff
@@ -269,21 +282,55 @@ def set_frequency(self, value=60):
         self.logger.debug('procmonitor: set frequency to ' + str(value))
         self.frequency = value
 
+    def get_system_stats(self, nw_protocols=['TCP']):
+        """
+        Run system monitoring
+        """
+        cmd = 'sar -rSub -n %s 1 1' % ','.join(nw_protocols)
```

Comment:
If sar isn't installed, this will fail. Maybe check if it is installed: if not, then don't run.

Reply:
@vstumpf test skip on sar command is not found.